pyccel/codegen/printing/pycode.py

```diff
@@ -215,59 +220,54 @@ def _print_PythonList(self, expr):
         args = ', '.join(self._print(i) for i in expr.args)
         return '['+args+']'
 
-    def _print_PythonLen(self, expr):
-        return 'len({})'.format(self._print(expr.arg))
-
-    def _print_PythonAbs(self, expr):
-        return 'abs({})'.format(self._print(expr.arg))
-
-    def _print_PythonMin(self, expr):
-        return 'min({})'.format(self._print(expr.args[0]))
-
-    def _print_PythonMax(self, expr):
-        return 'max({})'.format(self._print(expr.args[0]))
-
-    def _print_PythonSum(self, expr):
-        return 'sum({})'.format(self._print(expr.args[0]))
-
     def _print_PythonBool(self, expr):
         return 'bool({})'.format(self._print(expr.arg))
 
     def _print_PythonInt(self, expr):
         type_name = type(expr).__name__.lower()
         is_numpy  = type_name.startswith('numpy')
         precision = str(expr.precision*8) if is_numpy else ''
-        name = 'int{}'.format(precision)
-        if is_numpy:
-            self.insert_new_import(Import('numpy',name))
+        name = self._aliases.get(type(expr), expr.name)
+        if is_numpy and name == expr.name:
+            self.insert_new_import(
+                    source = 'numpy',
+                    target = expr.name)
```

Comment:
I don't understand why you added the condition `name == expr.name` and removed the argument `alias = name`... Can you please explain?

Reply:
If `name != expr.name` that means that the object is stored in the `_aliases` dictionary. The only way for it to be there is if there is already an import for this object, so we don't need to create an additional one